install/kubernetes/cilium/charts/operator/templates/deployment.yaml

```diff
@@ -36,6 +36,20 @@ spec:
         io.cilium/app: operator
         name: cilium-operator
     spec:
+{{- if or (ge .Capabilities.KubeVersion.Minor "14") (gt .Capabilities.KubeVersion.Major "1") }}
+      # In HA mode can't schedule 2 cilium-operator pods in the same node as
+      # they will clash each other.
```

Comment:
Nit: 

```suggestion
      # In HA mode, cilium-operator pods should not be scheduled on the same node as
      # they will clash each other.
```

Reply:
^ `must not`, not `should not` :-)

Reply:
also `clash each other` -> `clash with each other`